src/person/controllers/authenticate_controller.py

```diff
@@ -0,0 +1,118 @@
+import app.settings as api_settings
```

Comment:
ok i see... you think app should be api....
please...
change it to `api`... because this is what the module is lol